### ISSUE TITLE
8MUSES

### ISSUE THREAD

**Issue by trgswe** [RipMeApp/ripme#1653]
* Ripme version: 8MUSES
* Java version: 
* Operating system: 

* Exact URL you were trying to rip when the problem occurred:
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Detail the expected behavior here.

## Actual Behavior

Detail the actual (incorrect) behavior here. You can post log snippets or attach log files to your issue report.

**Comment by nsfwftfw** [comment]
Yes, I have the same issue. It looks like the web address needs to be updated to comics.8muses.com rather than just 8muses.com but I have no knowledge of how to program.
